code/game/objects/items/storage/belt.dm

```diff
@@ -636,50 +636,63 @@
 	item_state = "fannypack_yellow"
 	item_color = "yellow"
 
-/obj/item/storage/belt/sabre
-	name = "sabre sheath"
-	desc = "An ornate sheath designed to hold an officer's blade."
+/obj/item/storage/belt/sheath
 	icon_state = "sheath"
 	item_state = "sheath"
 	w_class = WEIGHT_CLASS_BULKY
+	var/hold_type
 
-/obj/item/storage/belt/sabre/ComponentInitialize()
+/obj/item/storage/belt/sheath/Initialize()

```

Comment:
this doesn't change anything, set it back to ComponentInitialize() (it's called at the end of Initialize anyway)

Reply:
this was just an accident, thank you code review